### ISSUE TITLE
Maps - modernize and clean up filter

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#1445]
Notes:
Changing the black bar that runs under the main drop down menu/header to add context and look more modern.

Updating filter to have a more seamless look with the above stats bar and making the word filter to Filters (also making this word the word that shows filters not the three line symbol). This filter bar should look like it is part of the overall site rather than seeming off as it does now and needs to be readable.

**Comment by wlcobb** [comment]
Ensure that the filter is mobile responsive also.

**Comment by wlcobb** [comment]
[CPI Review 2019-10-11_UCOMM.docx](https://github.com/uno-isqa-8950/uno-cpi/files/3720929/CPI.Review.2019-10-11_UCOMM.docx)

**Comment by wlcobb** [comment]
@manushah17 
8.	Maps pages/header
a.	Filter – make backgrounds consistent
b.	Make line red to be consistent with header line (in thickness too)
c.	Remove odd line under the word reset
d.	Change reset to white linked text with an underline rather than button
e.	Make the light gray box square and white not light gray
f.	Make the font black (all blacks including headers if not gray need to be the black listed above)
g.	Make the filters button consistent with other buttons (if red is not visible make white)
h.	Expand the header box for the maps pages and remove information buttons 
i.	Need to create an area for a snippet that gives context about the Map 

Here are the notes.  Make the filters button consistent with other buttons (if red is not visible make white)
Expand the header box for the maps pages and remove information buttons 
Need to create an area for a snippet that gives context about the Map 
Double in size and add context there.  Need to take info text and put it in text under title of map.  

 


Opacity 70% - 75% rich black.  Still use white text and allow to see map underneath it.
![maps_snippet](https://user-images.githubusercontent.com/42555289/67441845-1d91cb80-f5c4-11e9-843e-6d253f664d1f.PNG)
